### PR TITLE
Place old html markup into markup folder

### PR DESCRIPTION
**Pull request by piton13**

### CODE REVIEW

**Comment by piton13** [comment]
Fixed bugs and some unclear things.

File: client/html/blocks/menu/menu.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
             Settings
         </a>
 
-        <a href="/today.html" class="list-item menu__today">
+        <a href="#" class="list-item menu__today">
```
Comment (denich) [inline client/html/blocks/menu/menu.html:7]: Why not to change href to "/markup/today.html"?

File: client/html/blocks/menu/menu.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
             </div>
         </a>
 
-        <a href="/week.html" class="list-item menu__next-seven-days">
+        <a href="#" class="list-item menu__next-seven-days">
```
Comment (denich) [inline client/html/blocks/menu/menu.html:16]: Why not to change href to "/markup/week.html"?

File: client/html/blocks/menu/menu.html [hunk 3]
```diff
@@ -26,7 +26,7 @@
         </a>
         @@include('./project-list-item.html', { "project" : "education", "color": "green", "count" : "8","href" : "#" })
         @@include('./project-list-item.html', { "project" : "home", "color": "red", "count" : "3", "href" : "#" })
-        @@include('./project-list-item.html', { "project" : "work", "color": "blue", "count" : "2", "href" : "/project-work.html" })
+        @@include('./project-list-item.html', { "project" : "work", "color": "blue", "count" : "2", "href" : "#" })
```
Comment (denich) [inline client/html/blocks/menu/menu.html:29]: Why not to change href to "/markup/project-work.html"?

File: README.md [hunk 4]
```diff
@@ -41,3 +41,6 @@ Also restarts server while changes happens in server sources.
 + "__unused__" - This option warns when you define and never use your variables.
 
 For more info see: __http://jshint.com/docs/options/__
+
+##Route to old html markup
+Old html markup now placed on the next route: __http://localhost:3000/markup/__
```
Comment (denich) [inline README.md:46]: There is not page on this route. Need to create index.html file in markup folder to handle route. (Can be used the same as on main index route page project/page.html)

File: client/index.html [hunk 5]
```diff
@@ -1,7 +1,15 @@
 <!DOCTYPE html>
```
Comment (denich) [inline client/index.html:1]: There is still not angular content here (inserted by @@include) - should be removed

**Comment by denich** [comment]
This pull request can be merged (because those issues shouldn't be harmful).
Issues could be fixed in next pull request.

File: README.md (hunk 4 above)
Comment (piton13) [inline README.md:46]: I meant that each appropriate file we can find by the following path: http://localhost:3000/markup/<name_of_file>.html
where <name_of_file> is: ['today', 'week', 'project-work'].

Comment (denich) [inline README.md:46]: This a kind of recession. Previously we shouldn't place html to default route for view markup (I'm talking from user perspective)

**Pull request merged**